Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/User/UserGroup/ByUserGroup.tsx

```diff
@@ -36,7 +36,7 @@ import GroupForm from './GroupForm';
 const defaultSearchcols: Search.IField<Application.Types.iSecurityGroup>[] = [
     { label: 'Name', key: 'DisplayName', type: 'string', isPivotField: false },
     { label: 'Description', key: 'Description', type: 'string', isPivotField: false },
-    { label: 'Type', key: 'Type', type: 'string', isPivotField: false },
+    { label: 'Type', key: 'Type', type: 'enum', enum: [{ Label: 'AD', Value: 'AD' }, { Label: 'Database', Value: 'Database' }], isPivotField: false },
```

Comment:
Can these enums be pulled programmatically from openXDA? Also, this is missing azure type.

Reply:
Not sure where to pull them from. I'm not seeing them in any tables I would imagine they would be in but I'm also not entirely sure where to look.